client/Windows/wf_defaults.c

```diff
@@ -0,0 +1,118 @@
+/**
+ * FreeRDP: A Remote Desktop Protocol Implementation
+ *
+ * Copyright 2014 Marc-Andre Moreau <marcandre.moreau@gmail.com>
```

Comment:
well, I think you don´t need to have Marc-Andre in here, you´ve written that yourself, didn´t you ;)

Reply:
right!